mtp_bank_admin/apps/bank_admin/views.py

```diff
@@ -63,3 +65,11 @@ def download_bank_statement(request, receipt_date):
     })
 
     return response
+
+class Q_And_A_View(TemplateView):
+
+    template_name = "q_and_a_page/q_and_a_page.html"
+
+    
+
+
```

Comment:
Minor point, but you don't need this much space at the end of the file (or between the class name and the templae_name attribute) - one new line is enough.